README.md

```diff
@@ -217,3 +217,15 @@ So the only callables that would really be allowed would be:
 It seems a bit weird to use function for this use case (especially since they are not autoloadable). So we end up with static methods.
 
 Of course, we could allow the static methods to be in another class than the service provider, but it seems generally a good idea to have the services created in the service provider.
+
+## Puli integration
+
+This package provides a [Puli *binding type*](http://docs.puli.io/en/latest/discovery/getting-started.html): `container-interop/service-provider`. This binding type should contain the fully qualified class names of service providers.
+
+Packages implementing the `ServiceProviderInterface` can (optionally) bind the class name of the service provider to this binding type. This way, users consuming the service provider and using Puli can discover the service providers automatically.
+
+To register your service provider, simply use Puli's `bind` command:
+
+```sh
+puli bind --class Acme\\Foo\\MyServiceProvider container-interop/service-provider
+```
```

Comment:
I would go with something simpler, and more `can` than `should` to avoid any confusion.

> This package provides a [Puli *binding type*](http://docs.puli.io/en/latest/discovery/getting-started.html): `container-interop/service-provider`. Modules using Puli and implementing this standard can register service providers (fully qualified class names) through this binding type:

> …

> This way, frameworks or applications based on Puli can discover service providers automatically.